flowkit/arguments/arguments.go

```diff
@@ -145,14 +147,21 @@ func ParseWithoutType(args []string, code []byte, fileName string) (scriptArgs [
 			break
 		}
 
-		var value, err = runtime.ParseLiteral(argumentString, semaType, inter)
-		if err != nil {
-			return nil, fmt.Errorf(
-				"argument `%s` is not expected type `%s`",
-				parameterList[index].Identifier,
-				semaType.QualifiedString(),
-			)
+		var value cadence.Value
+
+		if isOptional && argumentString == "nil" {
```

Comment:
can we add this to the doc? I will always be unsure if I need to use `nil` or `null` or something else.

Reply:
sure I will add to this PR